cilium-cli/connectivity/check/context.go

```diff
@@ -816,7 +816,7 @@ func (ct *ConnectivityTest) initClients(ctx context.Context) error {
 	if ct.params.MultiCluster != "" {
 		multiClusterClientLock.Lock()
 		defer multiClusterClientLock.Unlock()
-		dst, err := k8s.NewClient(ct.params.MultiCluster, "", ct.params.CiliumNamespace)
+		dst, err := k8s.NewClient(ct.params.MultiCluster, "", ct.params.CiliumNamespace, "", []string{})
```

Comment:
There is a bit of inconsistency compared to the behavior of the other clustermesh commands. Essentially, here the `--as` and `--as-group` flag are not leveraged for the client targeting the remote cluster, while they are in the `clustermesh connect/disconnect` commands. I don't have a strong opinion on either approach, as both have their own advantages and disadvantages (e.g., you may have a consistent setup, or the same user/group may not be available in all clusters), but I'd be consistent (and mention the selected approach in the flag description). WDYT?

Reply:
thanks for the context, I was unfamiliar with how this was used and initialized. I added parameters to enable impersonating here as well. I think consistency is the easiest to understand approach here.